File: app/src/main/java/com/rahmitufanoglu/retrofitdemo/SplashActivity.java
package com.rahmitufanoglu.retrofitdemo;


import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;


public class SplashActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent networkIntent = new Intent(getApplicationContext(), SliderActivity.class);
        NetworkChangeReceiver networkChangeReceiver = new NetworkChangeReceiver();
        networkChangeReceiver.onReceive(this, networkIntent);
    }
}

File: app/src/main/java/com/rahmitufanoglu/retrofitdemo/SplashActivity.kt
package com.rahmitufanoglu.retrofitdemo


import android.app.Activity
import android.content.Intent
import android.os.Bundle

import com.rahmitufanoglu.retrofitdemo.receiver.NetworkChangeReceiver


class SplashActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val networkIntent = Intent(applicationContext, SliderActivity::class.java)
        val networkChangeReceiver = NetworkChangeReceiver()
        networkChangeReceiver.onReceive(this, networkIntent)
    }

}
